02_flow_compensation/exercises.m

```diff
@@ -62,79 +59,92 @@
+CM=set_translation_plot(CM, true);
+CM=set_noise_plot(CM, true);
+
+TASBEConfig.set('beads.beadModel','SpheroTech RCP-30-5A'); % Entry from BeadCatalog.xls matching your beads
+TASBEConfig.set('beads.beadBatch','Lot AA01, AA02, AA03, AA04, AB01, AB02, AC01, GAA01-R'); % Entry from BeadCatalog.xls containing your lot
+% Can also set bead channel if, for some reason, you don't want to use fluorescein as standard
+% This defaults to FITC as it is strongly recommended to use fluorescein standards.
+% TASBEConfig.set('beadChannel','FITC');
+
 CM = set_ERF_channel_name(CM, 'FITC-A'); % We'll explain this in the next exercise
 
 % Now let's read some files...
 raw = read_filtered_au(CM,[dosedata 'LacI-CAGop_C3_C03_P3.fcs']);
 compensated = readfcs_compensated_au(CM,[dosedata 'LacI-CAGop_C3_C03_P3.fcs'],0,1);
-% You should see an error: need to "resolve" the color model first!
+% You should see an error: need to "resolve" the color model first! Comment
+% out above line of code and run again.
 
-CM = resolve(CM);
+CM = resolve(CM); %resolve computes a ColorModel from all of the pointers that are set up in ColorModel.m
 
 % Ignore the warnings about finding only one bead peak and translation and 
-% mapping files: those have to do with ERF conversion, which we'll cover
+% mapping files (if received): those have to do with ERF conversion, which we'll cover
 % in our next exercise
 
-% Resolving the colormodel just produced a whol lot of files: let's take a
-% looks at the ones having to do with compensation.
-%
+% Resolving the colormodel just produced a whole lot of files within a folder titled "plots": let's take a
+% look at the ones having to do with compensation.
+
 % There is one "autofluorescence-C" graph for each channel.
-% This model covers both true autofluorescence and sensor noise, and it
-% built from the negative control.  These are fit against a gaussian
+% This model covers both true autofluorescence and sensor noise, and it is
+% built from the negative control. These are fit against a gaussian
 % model: the solid red line shows mean, and the dotted lines +/- 2 std.dev.
 % Notice that there is a small scatter of more highly fluorescent data points.
-% The autofluorescence model attempt to avoid being overly influenced by 
+% The autofluorescence model attempts to avoid being overly influenced by 
 % these outliers by dropping the top and bottom 0.1% of data.
 % Usually autofluorescence is insignificant, but in some cell lines it is strong.
-%
+
 % The "color-compensation-C-for-C" graphs show the color compensation model, computed
 % as a linear transform, after subtraction of autofluorescence.
 % The model is computed only from those subpopulations that are significantly
 % above autofluorescence.
 % 
 % The linear factors plus autofluorescence make up an affine transform that
-% compensates for spectral overlap.
+% compensates for spectral overlap. This is so that the entire cell
+% fluorescence contributes to the signal color.
```

Comment:
Not quite sure what you mean by this sentence: "This is so..."; can you try to clarify?

Reply:
done